src/api/spec/features/beta/webui/groups_spec.rb

```diff
@@ -42,12 +42,7 @@ def group_in_datatable(page, group)
   it 'create a group' do
     visit groups_path
 
-    if mobile?
-      within('#bottom-navigation-area') { click_link('Actions') }
-      within('#bottom-navigation-area') { click_link('Create Group') }
-    else
-      click_link('Create Group', href: group_new_path)
-    end
+    click_link(id: 'create-group')
```

Comment:
I'm curious, it didn't work anymore with `click_link('Create Group')`? It should since it's also matching on the title of a link.

Reply:
Let me check again

Reply:
Well, it works!